roles/robottelo/defaults/main.yml

```diff
@@ -3,5 +3,25 @@ robottelo_url: https://github.com/SatelliteQE/robottelo.git
 robottelo_directory: /root/robottelo
 robottelo_virtualenv_path: /root/robottelo/venv
 robottelo_version: master
-robottelo_test_arguments:
-  - tests/foreman/endtoend/test_api_endtoend.py
+robottelo_junit_file: results.xml
+robottelo_test_type: endtoend # one of [tier1|tier2|tier3|destructive|upgrade|endtoend]
```

Comment:
Can you only run one test type at a time? Or could this be an array of types?

Reply:
It is possible to run several test types at once, but I'm not sure I want to encourage that as I don't see any advantage in it, also it is not done in qa automation. I'd say it is better to run several times and have separate junit files per type.